recipe/meta.yaml

```diff
@@ -16,52 +15,61 @@ outputs:
   - name: oxigraph-server
     build:
       script:
-        - bash {{ RECIPE_DIR }}/build-oxigraph.sh  # [unix]
+        - bash {{ RECIPE_DIR }}/build-oxigraph.sh        # [unix]
         - call {{ RECIPE_DIR }}\bld-oxigraph-server.bat  # [win]
     requirements:
       build:
-        - {{ compiler('c') }}                    # [unix]
-        - {{ compiler('cxx') }}                  # [unix]
-        - {{ compiler('m2w64_c') }}              # [win]
-        - {{ compiler('m2w64_cxx') }}            # [win]
+        - {{ compiler("c") }}                    # [unix]
+        - {{ stdlib("c") }}                      # [unix]
+        - {{ compiler("cxx") }}                  # [unix]
+        - {{ compiler("m2w64_c") }}              # [win]
+        - {{ stdlib("m2w64_c") }}                # [win]
+        - {{ compiler("m2w64_cxx") }}            # [win]
         - m2w64-binutils                         # [win]
-        - {{ min_rust }}
-        - cargo-bundle-licenses
         - clangdev                               # [not osx]
         - llvmdev                                # [not osx]
+        - {{ compiler("rust") }}
+        - cargo-bundle-licenses
+      host:
+        - openssl
```

Comment:
If you set `--no-default-features --features rustls-native`, Oxigraph CLI should not require openssl

Reply:
When possible (e.g. all the `compiler` packages are working together at the time, not always a given), conda-forge generally tries to minimize vendoring and reliance on non-conda-forge-built packages. 

In this  instance, looks like `rustls` has more freshly-mitigated CVEs than `openssl`, so I'm inclined to roll with this... especially as it appears to be working, again.

Reply:
The trick is that to push rustls  CVE fixes, I might have to tag a new oxigraph version whereas to fix openssl, conda just needs to push the new version. So, using openssl might be better.

Reply:
Yep, got it in one. `openssl` migrations aren't fun (e.g. `postgres` was insisting it was doing things _right_, and made the 3.2 upgrade more difficult) but it's a well-understood thing, and is generally "push button" to measurably migrate across the whole channel when we _have_ to for `openssl x.y+1`... but for `openssl x.y.z+1` patch releases, nobody except the `openssl-feedstock` maintainers have to do anything.

Meanwhile, with Stuff In A Blob, we can _maybe_ infer vendored versions of `some-special-ssl` from `THIRD-PARTY.yml`, but...